README.md

```diff
@@ -1,24 +1,56 @@
-# README
+# Budget-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a mobile budget app built with rails and PostgreSQL. Users can easily truck their transactions.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on rails
+- postgresql
+- CSS
 
-* System dependencies
+## Video presentation
 
-* Configuration
+ > [Click here](https://www.loom.com/share/a1dd7a2d035d40e0b7d29bda9fc9d4b0?t=4)
 
-* Database creation
+## Live Demo
 
-* Database initialization
+ > [Click here](https://rails-capstone-budget-app.herokuapp.com/)
```

Comment:
Required

- Please fix the live demo link

Reply:
I am working on the heroku issue and I feel it will take me some time to fix it.